### ISSUE TITLE
Installation via Composer issue

### ISSUE THREAD

**Issue by craiglondon** [ethercreative/yii2-ip-ratelimiter#6]
I have attempted to add this extension to my project but I am getting a composer error.

```Problem 1
    - The requested package ethercreative/yii2-ip-ratelimiter ~2.0.0 exists as ethercreative/yii2-ip-ratelimiter[1.0, dev-master] but these are rejected by your constraint.
```

I have attempted to add this plugin via both ways 
php composer.phar require ethercreative/yii2-ip-ratelimiter "~2.0.0"
Adding "ethercreative/yii2-ip-ratelimiter": "~2.0.0" to my composer.json

**Comment by craiglondon** [comment]
I ended up doing ` composer require ethercreative/yii2-ip-ratelimiter "dev-master"` to get it installed, but that's not what the instructions specify

**Comment by archlemon** [comment]
Fixed by 9e3b3740bfab814ec2283dae7b68640e2b2e2337

**Comment by craiglondon** [comment]
Thanks!
